Makefile

```diff
@@ -63,9 +63,11 @@ go-grpc: clean .go-helpers-installed $(PROTO_OUT)
 fix-path: go-grpc
 	mv -f $(PROTO_OUT)/temporal/api/* $(PROTO_OUT) && rm -rf $(PROTO_OUT)/temporal
 
+# Copy the payload helpers
 copy-helpers:
-	# Copy the payload helpers
+	chmod +w $(PROTO_OUT)/common/v1/payload_json.go 2>/dev/null || true
 	cp $(PROTO_OUT)/internal/temporalcommonv1/payload_json.go $(PROTO_OUT)/common/v1/
+	chmod -w $(PROTO_OUT)/common/v1/payload_json.go
```

Comment:
I remove write permissions after copying so I stop accidentally editing the wrong file